combiner/combiner_test.go

```diff
@@ -0,0 +1,33 @@
+package combiner_test
+
+import (
+	"fmt"
+
+	"github.com/philippgille/gokv/combiner"
+	"github.com/philippgille/gokv/file"
+	"github.com/philippgille/gokv/gomap"
+)
+
+func ExampleNewStore() {
+	st, err := combiner.NewStore(combiner.Options{
+		Parallel: true,
+		Backends: []combiner.Backend{
+			combiner.MustBackend(gomap.NewStore(gomap.DefaultOptions)),
+			combiner.NewBackend(file.NewStore(file.DefaultOptions)),
+		},
+	})
+	if err != nil {
+		panic(err)
+	}
+	defer st.Close()
+
+	st.Set("something", 123)
```

Comment:
The linter doesn't like this, it failed the CI build: https://travis-ci.org/philippgille/gokv/builds/547971655#L1061

One option is to add an exception to the linter rules (I could do that later), but I think it would be better to also check for the error here, even if it's just an example.